src/translate.directive.ts

```diff
@@ -27,6 +28,13 @@ export class TranslateDirective implements AfterViewChecked, OnDestroy {
                 this.checkNodes(event.translations);
             });
         }
+
+        // subscribe to onDefaultLangChange event, in case the default language changes
+        if(!this.onDefaultLangChangeSub) {
+            this.onDefaultLangChangeSub = this.translateService.onDefaultLangChange.subscribe((event: DefaultLangChangeEvent) => {
+                this.checkNodes(event.translations);
```

Comment:
FAILED TESTS:
  TranslateDirective
    ✖ should update the DOM when the default lang changes
      Chrome 54.0.2840 (Mac OS X 10.12.1)
    Expected 'TEST' to equal 'This is a test'.
        at Object.<anonymous> (webpack:///tests/translate.directive.spec.ts?f2bc:115:0 <- config/spec-bundle.js:72492:73)

Seems like it is not updating the keys now any more.